Tasks/Common/packaging-common/util.ts

```diff
@@ -95,3 +95,31 @@ export function getProjectAndFeedIdFromInput(feedProject: string): any {
         projectId: projectId
     }
 }
+
+export enum LogType {
+    debug,
+    warning,
+    error
+}
+
+function log(message: string, logType: LogType) {
+    if (logType === LogType.warning) {
```

Comment:
Switch is a good one. If we add more log types we can change it to a switch but since there are only three options I'll leave it as is for now. Thanks!